libethcore/Precompiled.cpp

```diff
@@ -132,15 +132,31 @@ ETH_REGISTER_PRECOMPILED(modexp)(bytesConstRef _in)
 	return {true, ret};
 }
 
+namespace
+{
+	bigint expLengthAdjust(size_t _baseLength, size_t _expLength, bytesConstRef _in)
+	{
+		size_t const numConsideredBytes { min(_expLength, size_t(32)) };
```

Comment:
If using braces for initialization, use the same amount of spaces as with parentheses.
So `size_t const numConsideredBytes{min(_expLength, size_t(32))};`